src/plone/volto/patches.py

```diff
@@ -52,7 +53,9 @@ def construct_url(self, randomstring):
 
         registry = getUtility(IRegistry)
         settings = registry.forInterface(IVoltoSettings, prefix="volto", check=False)
-        settings_frontend_domain = getattr(settings, "frontend_domain", None)
+        settings_frontend_domain = os.environ["VOLTO_FRONTEND_DOMAIN"] or getattr(
```

Comment:
We don't want to require it.

```suggestion
        settings_frontend_domain = os.environ.get("VOLTO_FRONTEND_DOMAIN") or getattr(
```